.github/actions/bazel-test-all/action.yaml

```diff
@@ -50,7 +50,7 @@ runs:
           ${GITHUB_WORKSPACE}/ci/bazel-scripts/main.sh
           BAZEL_EXIT_CODE="$?"
 
-          if [ -n "$HONEYCOMB_API_TOKEN" ] && [ -f ./bazel-bep.pb ]; then
+          if [ -n "$BUILDEVENT_APIKEY" ] && [ -f ./bazel-bep.pb ]; then
```

Comment:
I don't think this is correct since [bazel/exporter is using `HONEYCOMB_API_TOKEN`](https://sourcegraph.com/github.com/dfinity/ic/-/blob/bazel/exporter/main.go?L113)

Reply:
I updated that as well! Do you still see issues?

Reply:
Oh I missed that. Sorry